summary_workflows/differential_usage/specification/example_files/assessment_out.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "_id": "APAeval:datasetA_TPR_toolA",
```

Comment:
Thanks! I have one question: is the benchmarking event (quantification, identification or differential usage) indicated anywhere in those IDs?

Reply:
Ohhh, good point... it is not.    
Let me think about how to introduce it, because "benchmarking event" is not a parameter passed to the workflow. I see how this might cause problems though, if in two events the same ground truth and the same metric names are used. On the other hand @AsierGonzalez said that the OEB DB will anyway give a new unique id to each object, and save this verbose id only as additional info. And when querying the DB we use the hardcoded event id (e.g. "OEBE0070000001" for identification) that is part of each summary workflow.
I'll anyway open a new issue to keep that point in mind. And maybe @txellferret also has some thoughts on this? Thanks @daneckaw !

Reply:
At the moment there are different metric names in different benchmarking events (e.g. `Sensitivity_10nt` etc in identification and `Sensitivity` in differential usage) so there shouldn't be any technical issues, but I see how this could be confusing at some point later on.